mappings/net/minecraft/client/gui/screen/CommandSuggestor.mapping

```diff
@@ -0,0 +1,95 @@
+CLASS net/minecraft/class_4717 net/minecraft/client/gui/screen/CommandSuggestor
+	CLASS class_464 SuggestionWindow
+		FIELD field_2764 suggestions Lcom/mojang/brigadier/suggestion/Suggestions;
+		FIELD field_2765 completed Z
+		FIELD field_2766 selection I
+		FIELD field_2767 mouse Lnet/minecraft/class_241;
+		FIELD field_2768 typedText Ljava/lang/String;
+		FIELD field_2769 inWindowIndex I
+		FIELD field_2771 area Lnet/minecraft/class_768;
+		METHOD <init> (Lnet/minecraft/class_4717;IIILcom/mojang/brigadier/suggestion/Suggestions;)V
+			ARG 2 x
+			ARG 3 y
+			ARG 4 width
+			ARG 5 suggestions
+		METHOD method_2370 mouseScrolled (D)Z
+			ARG 1 amount
+		METHOD method_2371 scroll (I)V
+			ARG 1 offset
+		METHOD method_2372 mouseClicked (III)Z
+			ARG 1 x
+			ARG 2 y
+			ARG 3 button
+		METHOD method_2373 draw (II)V
+			ARG 1 mouseX
+			ARG 2 mouseY
+		METHOD method_2374 select (I)V
+			ARG 1 index
+		METHOD method_2375 complete ()V
+		METHOD method_2376 discard ()V
+		METHOD method_2377 keyPressed (III)Z
+			ARG 1 keyCode
+			ARG 2 scanCode
+			ARG 3 modifiers
+	FIELD field_21596 BACKSLASH_S_PATTERN Ljava/util/regex/Pattern;
+	FIELD field_21597 client Lnet/minecraft/class_310;
+	FIELD field_21598 owner Lnet/minecraft/class_437;
+	FIELD field_21599 textField Lnet/minecraft/class_342;
+	FIELD field_21600 textRenderer Lnet/minecraft/class_327;
+	FIELD field_21601 slashRequired Z
+	FIELD field_21602 suggestingWhenEmpty Z
+	FIELD field_21603 inWindowIndexOffset I
+	FIELD field_21604 maxSuggestionSize I
+	FIELD field_21605 chatScreenSized Z
+	FIELD field_21606 color I
+	FIELD field_21607 messages Ljava/util/List;
+	FIELD field_21608 x I
+	FIELD field_21609 width I
+	FIELD field_21610 parse Lcom/mojang/brigadier/ParseResults;
+	FIELD field_21611 pendingSuggestions Ljava/util/concurrent/CompletableFuture;
+	FIELD field_21612 window Lnet/minecraft/class_4717$class_464;
+	FIELD field_21613 windowActive Z
+	FIELD field_21614 completingSuggestions Z
+	METHOD <init> (Lnet/minecraft/class_310;Lnet/minecraft/class_437;Lnet/minecraft/class_342;Lnet/minecraft/class_327;ZZIIZI)V
+		ARG 1 client
+		ARG 2 owner
+		ARG 3 textField
+		ARG 4 textRenderer
+		ARG 5 slashRequired
+		ARG 6 suggestingWhenEmpty
+		ARG 7 inWindowIndexOffset
+		ARG 8 maxSuggestionSize
+		ARG 9 chatScreenSized
+		ARG 10 color
+	METHOD method_23920 showSuggestions ()V
+	METHOD method_23921 mouseScrolled (D)Z
+		ARG 1 amount
+	METHOD method_23922 mouseClicked (DDI)Z
+		ARG 1 mouseX
+		ARG 3 mouseY
+		ARG 5 button
+	METHOD method_23923 draw (II)V
```

Comment:
Maybe `render` instead of `draw`? Screens use `render` (which is unobfuscated, I think?). Same for method_2373.

Reply:
Done.